articles/browser-extension-basics.md

```diff
@@ -0,0 +1,29 @@
+## Browser Extension Basics
+
+Useful Resources:
+
+- [MDN Anatomy of an Extension](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension)
```

Comment:
I'd suggest having the links at the bottom of the document.

```
<!-- Invisible List of References -->
[mozilla-anatomy-webext]: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension
```

Then you can use it with `[MDN Anatomy of an Extension][mozilla-anatomy-webext]`.

Reply:
Ah cool thanks! Pav said he'd fill me in on some of the Jekyll syntax